### PR TITLE
Fetching council summary from the litentry-graph

### PR DESCRIPTION
**Pull request by silva-fj**
- Refactoring `CouncilSummaryTeaser` to get the data from the litentry-graph
- Adding new components for `Account`'s

### CODE REVIEW

File: codegen.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: "https://api-gateway.litentry.io/graphql"
+schema: "https://graph.litentry.io/graphql"
```
Comment (ishrak-imam) [inline codegen.yml:2]: Shouldn't we also change it in the `LitentryApiContext`?

File: src/ui/components/Account/Account.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import type {ProxyAccount} from 'src/generated/litentryGraphQLTypes';
```
Comment (ishrak-imam) [inline src/ui/components/Account/Account.tsx:1]: Is it a replacement for the `AccountInfoInlineTeaser`?

Comment (silva-fj) [reply src/ui/components/Account/Account.tsx:1]: Yes, for now.

File: codegen.yml (hunk 1 above)
Comment (silva-fj) [reply codegen.yml:2]: yep

File: src/api/hooks/useCouncilSummary.tsx [hunk 3]
```diff
@@ -1,45 +1,53 @@
...
-    const [electionsInfo, bestNumber, primeMember] = await Promise.all([
-      api.derive.elections.info(),
-      api.derive.chain.bestNumber(),
-      api.query.council.prime(),
-    ]);
+const oneMinute = 60 * 1000;
 
-    const {termLeft, percentage} = getTermLeft(bnToBn(electionsInfo.termDuration || 0), bestNumber);
+export function useCouncilSummary() {
+  const {data, ...rest} = useQuery<{proxyCouncil: CouncilSummary}>(COUNCIL_SUMMARY_QUERY, {pollInterval: oneMinute});
```
Comment (ishrak-imam) [inline src/api/hooks/useCouncilSummary.tsx:47]: Do we need a polling interval for all dashboard queries?

Comment (silva-fj) [reply src/api/hooks/useCouncilSummary.tsx:47]: I think it's something we can do until we add subscriptions to the api

**Pull request merged**